src/test/java/com/telefonica/iot/cygnus/sinks/OrionMongoSinkTest.java

```diff
@@ -68,34 +69,79 @@
     private final String attrValue = "112.9";
     private final String attrMd = "[]";
     private final String serviceHeader = "vehicles";
-    private final String servicePathHeader = "4wheels";
-    private final String destination = "4wheelsCars";
+    private final String singleServicePathHeader = "4wheels";
+    private final String multipleServicePathHeader = "4wheels,4wheels";
+    private final String singleDestinationHeader = "4wheelsCars";
+    private final String multipleDestinationHeader = "4wheelsCars,4wheelsCars";
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-cygnus/commit/eaaf060cf9c8182340c6781a4d596499fa7adcf6